CSS/introduction.css

```diff
@@ -17,26 +17,51 @@
 }
 
 .main-content {
-    max-width: 75%;
+    /* max-width: 75%; */
+    width: 70%;
+    height: max-content;
     margin: var(--xs);
-    border: 2px solid black;
+    /* border: 2px solid black; */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
 }
 
 .main-content-landing-page {
     max-width: 48%;
     margin: var(--xs);
 }
 
+.main-content-example-box{
+    background-color: #f3f4f6;
```

Comment:
instead of using color codes or names directly, use variables

Reply:
okay, sure